Sources/FoundationEssentials/Date.swift

```diff
@@ -260,36 +249,12 @@ extension Date : CustomDebugStringConvertible, CustomStringConvertible, CustomRe
             return unavailable
         }
 
-        var info = tm()
-#if os(Windows)
-        var time = __time64_t(self.timeIntervalSince1970)
-        let errno: errno_t = _gmtime64_s(&info, &time)
-        guard errno == 0 else { return unavailable }
-#else
-        var time = time_t(self.timeIntervalSince1970)
-        gmtime_r(&time, &info)
-#endif
+        // We are casting to Int64 here which means we are rounding down.
+        // This is fine since we only need second granularity here for the description
+        let (year, month, day, hours, minutes, seconds) = Int64(self._time).utcDateFromTimestamp
 
-        // This allocates stack space for range of 10^102 years
-        // That's more than Date currently supports.
-        let bufferSize = 128
-        return withUnsafeTemporaryAllocation(of: CChar.self, capacity: bufferSize) { buffer in
-            guard let ptr = buffer.baseAddress else {
-                return unavailable
-            }
-
-            guard strftime(ptr, bufferSize, format, &info) != 0 else {
-                return unavailable
-            }
-
-            guard let result = String(validatingUTF8: ptr) else {
-                return unavailable
-            }
-
-            return result
-        }
+        return "\(year)-\(month)-\(day) \(hours):\(minutes):\(seconds) +0000"
```

Comment:
These string interpolations need to be padded with zeroes (0000-00-00 00:00:00 +0000)